### ISSUE TITLE
[med] Keyboard Navigation: Many Pages: Links have no visible focus indicator

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5455]
## Action Performed:
1. Using Chrome, open https://staging.new.expensify.com/
2. Press Tab key to focus the links 'the code' and 'open jobs'.
3. Note if the links have a visible focus indicator.
## Expected Result:
The links 'the code', and 'open jobs' must have a visible focus indicator.

## Actual Result:
The links 'the code', and 'open jobs' have no visible focus indicator.

HTML:
`<span class="css-901oao css-16my406" style="color: rgb(255, 255, 255); font-family: GTAmericaExp-Regular; font-size: 15px; line-height: 20px; text-decoration-line: underline;">the code</span>`

CSS:
`:focus:not([data-focusvisible-polyfill]) {
outline: none;
}`

Other occurrences:
1. Set password -> The links 'the code', and 'open jobs' have no visible focus indicator.

## Workaround:
No workaround

## Area issue was found in:
Landing page, Set password page

## Failed WCAG checkpoints
2.4.7

## User impact:
The keyboard users are not able to determine the current focus location.

## Suggested resolution:
Do not use outline:none unless an alternative highly visible focus outline is provided. See: https://www.w3.org/WAI/WCAG21/Techniques/general/G195.html

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**

https://user-images.githubusercontent.com/88733897/134548721-caa348f3-2586-40c7-8f45-3281a25a0e28.mp4

![Bug5180510_focus_visible_-_links_-_landing_page](https://user-images.githubusercontent.com/88733897/134548727-30b40d02-86e3-4a51-b2f0-e8b0904fc397.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
